### PR TITLE
task: introduce autoloading

### PR DESCRIPTION
**Pull request by benjaminkott**
With this patch, we are starting to introduce autoloading to the PHP
part of the application. With the introduction of composer, we can
make use of PSR-4 (https://www.php-fig.org/psr/psr-4/). This enables
us to avoid manually including PHP class files. Instead, we can import
them via namespaced call and the autoloader will provide the correct
resources for us.

All lib classes are now PSR-4 compatible and the new namespace
Photobooth was introduced. A new lib\boot.php file was introduced
that provides all the globally defined constants and loads the config.
PHP-Scripts that include the boot file will now have access to
autoloading, there is no need anymore for manual includes if the class/
package supports PSR-4.

The package phpmailer/phpmailer is now maintained via composer
and the git submodule was removed.

### CODE REVIEW

File: lib/src/Photobooth.php
Comment (andi34) [inline lib/src/Photobooth.php:1]: On `public static function serverOs()` use Environment class?

**Comment by andi34** [comment]
Readme need to be updated for PHP >= 8.2

**Comment by andi34** [comment]
install-photobooth.sh needs vendor/PHPMailer removed from submodules
https://github.com/PhotoboothProject/photobooth/blob/dev/install-photobooth.sh#L42

**Comment by benjaminkott** [comment]
> install-photobooth.sh needs vendor/PHPMailer removed from submodules

already removed.

**Comment by benjaminkott** [comment]
> Readme need to be updated for PHP >= 8.2

done

File: lib/src/Photobooth.php
Comment (benjaminkott) [reply lib/src/Photobooth.php:1]: more cleanups needed anyways, keep it as low as possible for now.

**Pull request merged**